### PR TITLE
fix: add z-index to BBackTop

### LINKED ISSUE #1970: Blue Up arrow hides behind reply and comments on mobile view
#### Expected behavior

On a mobile phone ( and desktop with mobile size), up arrow should always stay above the content and reply box, so its clickable

#### Actual behavior

Blue Up arrow on the right bottom of a mobile screen goes behind the comments and reply modal.

#### How to reproduce

- Log into your account.
- Find any post with at least 15 - 20 comments.
- Open the post and scroll to the bottom. This will load the comments.
- Scroll down so blue up arrow appears. 
- Now scroll up to see arrow go behind text and reply button.

#### Recording Of The Issues

https://youtu.be/I2ihhKqea-4

* Device : Samsung S8
* System Operating : 8.0.0 Oreo
* Browser : Google Chrome ( 66.0.3359.158 Version)

### PR DESCRIPTION
**Pull request by Sekhmet**
Fixes #1970 

### Changes

Summary of changes you made.

* Add `z-index` to `BBackTop`

### Test plan

* [x] Go to: http://localhost:3000 and login.
* [x] Find post with 15+ comments.
* [x] Scroll down so comments load.
* [x] Scroll up so BBackTop would appear above reply form

### Demo 

Before:
image

After:
image

### CODE REVIEW

**Comment by bonustrack** [comment]
I cant reproduce this issue on staging do you @Sekhmet ?

**Comment by Sekhmet** [comment]
I can.

**Comment by bonustrack** [comment]
Weird, anyway both are working fine for me

File: src/client/components/BBackTop.less [hunk 1]
```diff
@@ -7,6 +7,7 @@
   height: 0px;
   bottom: 60px;
   right: 20px;
+  z-index: 1500;
```
Comment (jm90m) [inline src/client/components/BBackTop.less]: might want to use a variable from here - https://github.com/busyorg/busy/blob/a538b8d4f3a03c4cff15f2e5c39c507783be4d51/src/client/styles/custom.less#L95 

Or create new one.

**Pull request merged**